.github/workflows/deploy-main.yml

```
@@ -0,0 +1,47 @@
+name: Deploy Main to GitHub Pages
+
+on:
+  push:
+    branches: [main]
```

```suggestion
    branches:
      - main
      - x86_64
```
It would be nice for us to serve the `x86_64` version as well. For that, I'll change the `baseUrl` to be `.../x86_64` in `_config.yaml`. Or rather, can you serve it directly by pushing to a `x86_64/` folder on the `gh-pages` branch?

---

That would be a different workflow that pushes `x86_64` to `gh-pages/x86_64`

---

Alright. Please add it if you have the time